cmd/migration-manager/internal/cmds/source_test.go

```diff
@@ -22,16 +22,13 @@ import (
 	"github.com/FuturFusion/migration-manager/internal/testing/queue"
 )
 
-var (
-	additionalRootCertificate tls.Certificate
-	vCenterSimulator          *simulator.Server
-)
+var vCenterSimulator *simulator.Server
```

Comment:
The whole purpose of this PR is to move the logic to test the connectivity to a source (and a target) to the service. Why should in this case the tests continue to life in the migration-manager CLI package. This is no longer the migration-manager CLIs concern. This whole test should be move to the place, where we have the logic to call out to a source.